mkFit/fittestMPlex.cc

```diff
@@ -24,6 +25,7 @@
 
 #include <iostream>
 #include <memory>
+#include <mutex>
```

Comment:
Why was this added?  (There's already an include of mutex inside "ifndef NO_ROOT", and the only use of a mutex is also inside "ifndef NO_ROOT")